spec/models/topic_spec.rb

```diff
@@ -505,6 +505,17 @@ def build_topic_with_title(title)
           expect(topic.reload.allowed_users).to_not include(another_user)
           expect(Post.last.action_code).to eq("removed_user")
         end
+
+        context "from a muted user" do
+          before { MutedUser.create(user: another_user, muted_user: user) }
```

Comment:
I always prefer to use `create!` since it'll throw an error and point us at the problem when the creation fails.